docs/source/requirements-by-module/plot-description/slope-type/index.adoc

```diff
@@ -1,5 +1,15 @@
 ===== Slope type Observation
 
+include::slope-type_feature-type.adoc[]
+
 include::slope-type_result-value.adoc[]
 
+include::slope-type_simple-result.adoc[]
+
+include::slope-type_site-visit.adoc[]
+
+include::slope-type_used-procedure.adoc[]
+
+include::slope-type_value-type.adoc[]
+
```

Comment:
In the file `src/asciidoc/templates.py`, can you please append the following line:

```
// Do not edit this file directly.
```

to line 29.